applicationinsights/TelemetryClient.py

```diff
@@ -36,7 +35,7 @@ def context(self):
         Returns:
             :class:`channel.TelemetryChannel`. the context instance.
         """
-        return self._context
+        return self._channel.context
```

Comment:
This change breaks scenario when you may want to have multiple telemetry clients with the different set of properties. And one channel that has a few extra properties that applicable to ALL clients. So as I mentioned it would be better fix to modify how `write` method works - it should merge channel's context and telemetry client's context

Reply:
I'm a bit lost here on how you would achieve your scenario in the existing code base. Can you give a small code example? Then I can use that as basis for a test case.

Reply:
I want something like this to be possible:

``` python
channel = channel.TelemetryChannel()
channel.context.properties['foo'] = 'bar'
firstClient = TelemetryClient(key, channel)
firstClient.context.properties['flightId'] = 'first flight telemetry'

secondClient = TelemetryClient(key, channel)
secondClient.context.properties['flightId'] = 'second flight telemetry'

firstClient.trackTrace('Test event 1')

secondClient.trackTrace('Test event 2')
```

first and second events should have different `flightID`